src/widgets/source/ui/CalibratorsPlotWidget.cpp

```diff
@@ -725,8 +724,9 @@ namespace SmartPeak
                   user_excluded_points_.erase(std::remove(user_excluded_points_.begin(), user_excluded_points_.end(), sample_and_serie), user_excluded_points_.end());
                   if (selected_action_ == EActionFitCalibration)
                   {
-                    recomputeCalibration();
-                  }                }
+                    recomputeFitCalibration(serie_name);
```

Comment:
It looks like I did not catch this last time but I think it should be "series" and not "serie" for the variable names?

Reply:
it's about only one serie (singular). it looks ok for me unless maybe i didn't get your comment correctly

Reply:
"series" is both singular and plural.  Anyway, very minor.  We can fix it in a subsequent PR.